CONTRIBUTING.md

```diff
@@ -1,19 +1,65 @@
-Code formatting
-===============
+# How to contribute
 
-Go provides a tool, `go fmt`, which facilitates a standardised format to Go 
-source code. SQLair has two additional policies:
+Thanks for your interest in SQLair! Contributions like yours make good projects
+great.
 
-Imports
--------
+When contributing, you must abide by the [Ubuntu Code of
+Conduct](https://ubuntu.com/community/ethos/code-of-conduct).
 
-Import statements are grouped into three sections: 
-- standard library 
-- third party libraries 
-- SQLair imports 
+## Licence and copyright
 
-The tool "go fmt" can be used to ensure each group is alphabetically sorted, 
-for example:
+By default, all contributions to SQLair are made under the Apache 2.0 licence.
+See the [licence](LICENSE) in the SQLair GitHub repository for details.
+
+All contributors must sign the [Canonical contributor licence
+agreement](https://ubuntu.com/legal/contributors), which grants Canonical
+permission to use the contributions. The author of a change remains the
+copyright owner of their code (no copyright assignment occurs).
+
```

Comment:
For docs at Canonical the license is 

© 2024 CC-BY-SA, Canonical Ltd. 

See, e.g., the footer of https://juju.is/docs/juju

Reply:
I'll add a bit saying that this license is used for the docs (assuming that is the case?).